go/internal/sidecar/ws_bridge.go

```
@@ -0,0 +1,344 @@
+		if json.NewDecoder(resp.Body).Decode(&v) != nil {
+			protocolWSError(w, "Invalid JSON payload in upstream response")
+			return
+		}
+		sendJSONEvents(w, v)
+		return
+	}
+	stream := newWSSSEBlockReader(bufio.NewReader(resp.Body), maxWSFrameBytes)
+	first := true
+	for {
+		block, e := stream.Next()
+		if e == io.EOF {
```

golangci-lint (2.13.2)

[error] comparing with == will fail on wrapped errors. Use errors.Is to check for a specific error

(errorlint)